app/models/article.rb

```diff
@@ -29,15 +31,8 @@ class Article < ActiveRecord::Base
     where(evidence: true)
   }
 
-  ########### Slug change on update ################
-  attr_writer :use_slug
-
-  def use_slug
-    @use_slug || true
-  end
-
-  def should_generate_new_friendly_id?
-    slug.blank? || title_changed? if use_slug == "1"
+  default_value_for :date do
+    Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.